go/src/koding/kites/config/konfig.go

```diff
@@ -135,6 +136,7 @@ func NewKonfig(e *Environments) *Konfig {
 		KontrolURL:         Builtin.Endpoints.Kontrol,
 		KloudURL:           Builtin.Endpoints.Kloud,
 		TunnelURL:          Builtin.Endpoints.TunnelServer,
+		RemoteURL:          Builtin.Endpoints.RemoteAPI,
```

Comment:
This is to fix hacks like this one:

https://github.com/koding/koding/pull/9786/files#diff-58db80dbc542b0ed622120b6235087c3R64